packages/teleport/src/components/SelectFilter/SelectFilter.tsx

```diff
@@ -0,0 +1,328 @@
+/**
+ * Copyright 2021 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { components } from 'react-select';
+import { Flex, Text, ButtonBorder, ButtonIcon, Box } from 'design';
+import { Close, Add } from 'design/Icon';
+import Select, { Option } from 'shared/components/Select';
+
+export default function SelectFilter({
+  applyFilters,
+  appliedFilters,
+  filters,
+  mb = 3,
+}: Props) {
+  const selectWrapperRef = React.useRef(null);
```

Comment:
Nit: some React features are imported like this `React.useRef` and some just as `useState`, for consistency I'd stick to one style (for me `useState` looks better)